megatron/megatron/arguments.py

```diff
@@ -1251,7 +1251,7 @@ def _add_distributed_args(parser):
                        help='overlap pipeline parallel communication with forward and backward chunks',
                        dest='overlap_p2p_comm')
     group.add_argument('--distributed-backend', default='nccl',
-                       choices=['nccl', 'gloo'],
+                       choices=['nccl', 'gloo', 'xccl', 'mccl'],
```

Comment:
First, `xccl` has bee removed. Second, why not add these change in the `mthreads/patches/arguments.py` instead of changing this original src code.